src/frontend/src/pages/PPQFormPage/PPQFormPage.tsx

```diff
@@ -90,10 +95,10 @@ const PPQFormPage = () => {
     const requestBody: IPPQForm = {
       title: title,
       ministry: ministry,
-      description,
+      description: description,
       proposedStartDate: startDate,
-      piaType,
-      delegatedReviewType,
+      piaType: piaType,
+      delegatedReviewType: delegatedReviewType,
```

Comment:
This shouldn't have failed at build. Looks like a legit syntax 🤔

Reply:
It had to do with the state declarations needing to have specified types i.e.: `useState<PiaTypeEnum | string>(PIATypes[0].value)` instead of simply, `useState(PIATypes[0].value)`